meta/main.yml

```diff
@@ -12,6 +12,7 @@ galaxy_info:
     - name: Ubuntu
       versions:
         - bionic
```

Comment:
Maybe let's remove this line from the metadata file to indicate support for Ubuntu 18.04 is dropped

Reply:
Sorry, I actually meant the whole `Ubuntu` block